src/createServer.js

```diff
@@ -1,3 +1,55 @@
+/* eslint-disable max-len */
 // Write code here
 // Also, you can create additional files in the src folder
 // and import (require) them here
+const http = require('http');
+const convertToCase = require('./convertToCase').convertToCase;
+
+function createServer() {
+  const server = http.createServer();
+
+  server.on('request', (req, res) => {
+    const urlParts = req.url.split('?');
+    const textToConvert = urlParts[0].slice(1);
```

Comment:
you could rename this variable to originalText and use it by itself in data object